integration-tests/manual-api/test.fake.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
```

Comment:
ℹ️ these are tests run by the "fake testing framework"

Reply:
Can the folder be moved maybe to `ci`? `manual-api` really seems generic.

Reply:
I moved it to `integration-tests/ci-visibility/test-api-manual`